x-pack/platform/plugins/shared/dataset_quality/public/components/dataset_quality_details/overview/quality_summary_cards/card.tsx

```diff
@@ -19,7 +19,7 @@ export function Card({
 }: {
   isDisabled?: boolean;
   isSelected?: boolean;
-  title: string;
+  title: React.ReactNode;
```

Comment:
We would need to change `data-test-subj={datasetQualityDetailsSummaryKpiValue-${title}}` for something else right? This worked before because title was a string